src/cirrus/test.py

```diff
@@ -49,10 +84,21 @@ def main():
     Execute test command
     """
     opts = build_parser(sys.argv)
-    if opts.suite is not None:
-        nose_test(opts.suite)
-    else:
-        nose_test('default')
+    config = load_configuration()
+    mode = config.test_mode(opts.suite)
+    if opts.mode:
+        mode =  opts.mode
+
+    # backwards compat: default to nosetests
+    if mode is None:
+        mode = 'nosetests'
```

Comment:
Yeah, but since the argparse option overrides the conf file, that would give me a condition where the argparse option always overrides any conf file option. So I kept it a bit spread out so that it was obvious what was happening